sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/ControlFlowGraph/ControlFlowGraphTest.cs

```diff
@@ -1595,6 +1595,72 @@ public void Cfg_ConditionalAccessNested()
             assignment.SuccessorBlocks.Should().OnlyContain(exitBlock);
         }
 
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_ConditionalAccess_Coalesce()
+        {
+            var cfg = Build("if(aObj?.booleanVal ?? false)");
+            VerifyCfg(cfg, 7);
+
+            var branchBlock1 = cfg.EntryBlock as BinaryBranchBlock;
+            var blocks = cfg.Blocks.ToList();
+            var aObj = blocks[1];
+            var branchBlock2 = blocks[2] as BinaryBranchBlock;
+            var coalesceBranchingNode = blocks[3];
+            var flse = blocks[4] as BinaryBranchBlock;
+            var identifierName = blocks[5];
+            var exitBlock = cfg.ExitBlock;
+
+            branchBlock1.SuccessorBlocks.Should().OnlyContainInOrder(branchBlock2, aObj);
```

Comment:
for `BinaryBranchBlock` it's more readable to use `TrueSuccessorBlock.Should().Be(x)` and `FalseSuccessorBlock.Should().Be(y)`